Maps/ArchipelagoCampaign/WoL/ap_media_blitz.SC2Map/MapScript.galaxy

```diff
@@ -960,6 +960,19 @@ bool gt_Init03Units_Func (bool testConds, bool runActions) {
     gv_statueBullhorns[4] = UnitFromId(213);
     gv_statueBullhorns[5] = UnitFromId(217);
     gv_statueBullhorns[6] = UnitFromId(215);
+    if ((libABFE498B_gv_aP_Triggers_Option_overridePlayerRace == "Zerg")) {
+        UnitSetPropertyFixed(gv_odin, c_unitPropLifeRegen, 1.8);
+        UnitBehaviorAdd(gv_odin, "AP_ZergOdin", gv_odin, 1);
+    }
+    else {
+        if ((libABFE498B_gv_aP_Triggers_Option_overridePlayerRace == "Prot")) {
+            UnitSetPropertyFixed(gv_odin, c_unitPropLifeMax, 2000.0);
+            UnitSetPropertyFixed(gv_odin, c_unitPropLife, 2000.0);
+            UnitSetPropertyFixed(gv_odin, c_unitPropShieldsMax, 500.0);
+            UnitSetPropertyFixed(gv_odin, c_unitPropShields, 500.0);

```

Comment:
This can erase Vanadium Plating HP boost or all Terran mech HP boost (test with player having auto unlock W/A and having those stuff). Odin doesn't start the mission always with 2500 HP due to upgrades